cli/bin/simple.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 BIN=`dirname "$0"`
 BASE=$BIN/..
-java -Xmx10g -cp "$BASE/config:$BASE/target/classes:$BASE/target/*" \
-edu.stanford.futuredata.macrobase.runner.SimpleRunner "$@"
+java -Xmx10g -cp "$BASE/config:$BASE/target/classes:$BASE/../lib/target/classes:$BASE/target/*" \
```

Comment:
This probably isn't the right way to do things, but for some reason the NormalDist LUT resource file in lib was causing a FileNotFoundException when I ran the pipeline from CLI, so I manually added it to the classpath here.